build.gradle

```diff
@@ -9,24 +9,18 @@ buildscript {
             name = 'minecrell'
             url = 'http://repo.minecrell.net/releases'
         }
-        maven {
-            name = 'javalite'
-            url = 'http://repo.javalite.io'
-        }
     }
     dependencies {
         classpath 'net.minecrell:VanillaGradle:2.0.3_1'
         classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.4'
         classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7'
         classpath 'net.minecraftforge.gradle:ForgeGradle:2.3-SNAPSHOT'
-        classpath 'org.javalite:activejdbc-gradle-plugin:1.4.13-SNAPSHOT'
```

Comment:
Just change the version number to 2.2 instead of removing it simply since VC needs active jdbc instrumentation. Please refer to [this](http://javalite.io/instrumentation).